linux/src/threadpool_linux.c

```diff
@@ -97,6 +98,31 @@ typedef struct THREADPOOL_TAG
 
 THANDLE_TYPE_DEFINE(THREADPOOL);
 
+#define TIMER_TABLE_SIZE 100
```

Comment:
Looking at the changes it seems like we can only have 100 times at any one time.  What happens when we go over the 100 timer mark (which can happen really quick in the software)?  Do we have any failure mechanism as in do we fail the timer create?  Should we?

Reply:
done increased the size of timer table and added retry for 2 times as discussed : )